### PR TITLE
This patch allows htmlwidgets to display in pkgdown sites.

### PR DESCRIPTION
**Pull request by dmurdoch**
It requires the corresponding branch of downlit and pkgdown to have been installed:

    remotes::install_github("dmurdoch/downlit@rglpatch")
    remotes::install_github("dmurdoch/pkgdown@rglpatch")

PR's have been sent to those two packages.

### CODE REVIEW

**Comment by jcheng5** [comment]
Upstream PRs:
* https://github.com/r-lib/downlit/pull/78
* https://github.com/r-lib/pkgdown/pull/1548

**Comment by hadley** [comment]
Just merged the pkgdown fix, so once the remotes are updated this should be good to merge.

File: DESCRIPTION [hunk 1]
```diff
@@ -24,9 +24,12 @@ Imports:
...
     knitr (>= 1.8),
+    downlit,
+    pkgdown,
     rmarkdown,
     testthat
 Enhances: shiny (>= 1.1)
 URL: https://github.com/ramnathv/htmlwidgets
 BugReports: https://github.com/ramnathv/htmlwidgets/issues
 RoxygenNote: 7.1.1
+Remotes: dmurdoch/downlit@rglpatch, dmurdoch/pkgdown@rglpatch
```
Comment (wch) [inline DESCRIPTION:35]: Have these PRs been merged? If so, the Remotes can be updated to point to the main repositories.

**Comment by dmurdoch** [comment]
Yes, they are merged into the devel versions on Github, but not yet on CRAN.  I'll change the Remotes field.

**Comment by dmurdoch** [comment]
The patches to pkgdown and downlit were merged long ago, but a later patch to pkgdown broke things again.  The PR to fix the breakage (https://github.com/r-lib/pkgdown/pull/1719) has been languishing there since June.  Currently I'm recommending these versions to get things to work:

    remotes::install_github(c( "r-lib/downlit",
                          "dmurdoch/pkgdown@issue1689"))

It's quite frustrating how long it's taken to get these PRs merged (including this one, which has been here since March).

**Comment by hadley** [comment]
I'm going to be focussing the majority of my time on pkgdown and friends for the next few weeks, so I'll push this along as needed.

**Comment by hadley** [comment]
You can close this; I've incorporated the code into downlit and pkgdown.

**Pull request closed without merging**